tool/generate_example.dart

```diff
@@ -5,6 +5,7 @@ import 'package:recase/recase.dart';
 
 const Map<String, String> nameAdjustments = {
   "500px": "fiveHundredPx",
+  "360-degrees": "threeSixtyDegrees",
```

Comment:
This change is defenitely necessary, but I would prefer "threeHundredSixtyDegrees", to keep the naming consistent with the above "fiveHundredPx".
```suggestion
  "360-degrees": "threeHundredSixtyDegrees",
```